### PR TITLE
Added Solution to #4

### PR DESCRIPTION
**Pull request by ghost**
# Frequency of Characters

Implementation for Issue #4 in TypeScript.

## Type of change

Please delete options that are not relevant.

- [X ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ X] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [X ] I have made corresponding changes to the documentation
- [ X] My changes generates no new warnings

### CODE REVIEW

**Comment by sakshamarora1** [comment]
Thank you for opening a pull request @Kwehdev, this is great!

We are trying to follow the same style while adding snippets. Please refer to this [example](https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/snippets/python/tablularformatting.md). The code should come under the `Code` sub-heading and any other important information can be added under the `Description` sub-heading. [Contributing Guidelines](https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/CONTRIBUTING.md)

File: snippets/typescript/frequencyOfCharacters.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Count amount of characters in a given string, output to .txt file.
```
Comment (sakshamarora1) [inline snippets/typescript/frequencyOfCharacters.md]:
```suggestion
# Count frequency of unique characters in a string
```

**Comment by ghost** [comment]
> Thank you for opening a pull request @Kwehdev, this is great!
> 
> We are trying to follow the same style while adding snippets. Please refer to this [example](https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/snippets/python/tablularformatting.md). The code should come under the `Code` sub-heading and any other important information can be added under the `Description` sub-heading. [Contributing Guidelines](https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/CONTRIBUTING.md)

Ah, my apologies. Amended as requested, hopefully that satisfies the guidelines.

**Review by sakshamarora1 — APPROVED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@Kwehdev Great! Thanks for your contribution!

**Pull request merged**